src/core/execute.c

```diff
@@ -4033,6 +4036,7 @@ static void strv_fprintf(FILE *f, char **l) {
 }
 
 void exec_context_dump(const ExecContext *c, FILE* f, const char *prefix) {
+        char buf_timespan[FORMAT_TIMESPAN_MAX];
```

Comment:
unless we share this buffer with other fields let's move it to the innermost block where we need it, i.e. into the `if (c->log_rate_limit_interval_usec > 0)` block